structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,135 @@
+4. Once the Collaborators informally agree that enough information has been gathered, the vote can be created in a private channel which cannot be seen by the Invitees, with the option to approve or deny, according to the process defined in this document. The Invitees should be informed the vote has begun.
+5. After the poll is closed, the vote is considered passed if more than 75% of the votes cast are positive. The discussion channel must be archived and kept read-only.
+- If the vote has passed: the Invitees are elevated to the rank of Collaborators. Internal discussions can be purged first if needs be. The community is added to any listing that may exist.
+- If the vote is rejected: enough time is given to the Invitees to acknowledge the result and leave the server. Managers are free to kick Invitees if necessary.
+
+# Process: removing a community
+
+The Managers can decide to start the process of removing a community from the program, due to external events or shared concern between communities. The process is defined as follows:
+1. The Managers can temporarily remove the community from the program by replacing roles of the affected members with the Invitee role, if they believe it is necessary to protect the other communities. The Managers are free to communicate the reason to the community, but it is not required.
+2. The Managers shall write a message clearly explaining the concerns, with an attached thread. In any case, the affected communities cannot have access to this thread. A channel can be set up to allow discussions and clarifications.
+3. Once the Collaborators informally agree that everyone has had enough time to read the message and ask questions, the vote can be created, with the option to approve or deny, according to the process defined in this document. No veto can be used in this vote.
+4. After the poll is closed, the vote is considered passed if more than 50% of the votes cast are positive. The discussion channel must be archived and kept read-only.
+- If the vote has passed: the community is removed from the program and from any listing that may exist.
+- If the vote is rejected: the community is reintegrated to the program, and roles are given back if necessary. Internal discussions can be purged first if needs be.
+
+# Rationale and alternatives
+
+The Quilt Community Collab program existed prior to this document. This document tries to summarize how the server has been working so far while formalising some aspects that were still left up in the air.
+
+More importantly, three polls have been held inside the server, with the following results:
+
+- When voting for changes on who is or isn't part of the CC network, how should we count votes?
+    - 10 answered: One vote per community: each community will have to write down a formal process on which they decide to vote for or against changes. This process must be democratic and fair.
```

Comment:
If we're requiring a documented, formal voting process for each community (which I don't think we actually need), it should be provided as part of a community's application. The RFC isn't clear on this, however.